bin/build_pipeline_heka.sh

```diff
@@ -52,6 +58,40 @@ fi
 
 source build.sh
 
+echo 'Installing lua-geoip libs'
+cd $BASE/build
+if [ ! -d lua-geoip ]; then
+    # Fetch the lua geoip lib
+    git clone https://github.com/agladysh/lua-geoip.git
+    cd lua-geoip
+    # from 'make.sh'
+    gcc -O2 -fPIC -I${LUA_INCLUDE_PATH} -c src/*.c -Isrc/ -Wall --pedantic -Werror --std=c99 -fms-extensions
+
+    SO_FLAGS="-shared -fPIC"
+    UNAME=$(uname)
+    case $UNAME in
+    Darwin)
+        echo "Looks like OSX"
+        SO_FLAGS="-bundle -undefined dynamic_lookup"
+        ;;
+    *)
+        echo "Looks like Linux"
+        # Default flags apply.
+        ;;
+    esac
+
+    gcc $SO_FLAGS database.o city.o -o city.so
+    gcc $SO_FLAGS database.o country.o -o country.so
+    gcc $SO_FLAGS database.o lua-geoip.o -o geoip.so
+    cd -
+fi
+
+cd $BASE/build/heka/build
+cp $BASE/build/lua-geoip/geoip.so heka/modules/
+mkdir -p heka/modules/geoip
```

Comment:
If you make this first you can build directly into the target directory and not have to copy